File: mylib-system-base-manager/mylib-system-base-manager-logic-jpa/src/main/java/com/zy/mylib/system/entity/CodeMap.java
package com.zy.mylib.system.entity;

import com.zy.mylib.data.jpa.NameCodeEntity;

import javax.persistence.Entity;
import javax.persistence.Table;

/**
 * @author ASUS
 */
@Entity
@Table(name = "sys_codemap")
public class CodeMap extends NameCodeEntity {
  private static final long serialVersionUID = -3532091040124604358L;

  @Override
  public String description() {
    return "字典分类";
  }
}

File: mylib-modules/mylib-config-module/src/main/kotlin/com/zy/mylib/sys/config/entity/CodeMap.kt
package com.zy.mylib.sys.config.entity

import com.zy.mylib.mongo.model.BaseMongoModel
import javax.persistence.*
import org.springframework.data.mongodb.core.mapping.*

/**
 * 代码集
 * @author 代码生成器
 */
@Document("sys_codemap")
class CodeMap: BaseMongoModel() {
  /**
   * 代码集编号
   */
  @Column
  var code: String? = null
  /**
   * 代码集名
   */
  @Column
  var caption: String? = null
}
